trinity/config.py

```diff
@@ -96,6 +98,20 @@ def __init__(self,
         elif nodekey is not None:
             self.nodekey = nodekey
 
+        self.logfile_path = get_logfile_path(self.data_dir)
```

Comment:
It'd be nice if users could specify the path in the `__init__` and if it isn't provided then it falls back to this value similar to how `data_dir` gets set.